client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -75,8 +69,8 @@ const BusinessPlanDetails = ( {
 					'Schedule a Quick Start session with a Happiness Engineer to set up ' +
 						'your site and learn more about WordPress.com.'
 				) }
-				buttonText={ i18n.translate( 'Schedule a session' ) }
-				href={ `/me/concierge/${ selectedSite.slug }/book` }
+				buttonText={ i18n.translate( 'Purchase a session' ) }
```

Comment:
Since this new string isn't translated yet, it'll result in non-English users seeing it in English, while everything else is in their chosen language.
<img width="718" alt="Screen Shot 2020-07-29 at 2 38 23 PM" src="https://user-images.githubusercontent.com/690843/88856454-5ee58600-d1a9-11ea-89aa-5a98b5d00ca5.png">

I've add the "string freeze" label which will trigger the string to get sent out for translation before the PR gets merged, but that will still take over a week.

Do you think it makes sense to just hide this whole block for non-English users for now?

Reply:
That makes sense. 
Used `hasTranslation` to check for translations but it was always returning false for **en** local.